src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRInfoViewController.swift

```diff
@@ -70,7 +70,7 @@ class ExposureSubmissionQRInfoViewController: DynamicTableViewController {
 
 		tableView.register(
 			UINib(nibName: String(describing: DynamicAcknowledgementCell.self), bundle: nil),
-			forCellReuseIdentifier: ReuseIdentifiers.acknowledgement.rawValue
+			forCellReuseIdentifier: ReuseIdentifiers.legal.rawValue
```

Comment:
If think if the case was changed to .legal the Cell class it self should be called something like 'DynamicLegalCell' ?

Reply:
The cell is renamed but in this context I find 'acknowledgement' more appropriate.